build/buildkite-build.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -xe
+
+cd `dirname $0`/..
+
+###
+### BUILD
+###
+
+/usr/local/bin/sbt                  \
+  -jvm-opts build/buildkite-jvmopts \
+  -no-colors                        \
+  -Docs3.skipDependencyUpdates      \
+  headerCheck                       \
+  test:headerCheck                  \
+  scalastyle                        \
+  compile
+
+###
+### TEST
+###
+
+# Start a new Postgres container for this build
+# TODO: read postgres version from the build
+CID=`docker run --detach --publish 5432 postgres:9.6.0`
+
+# Add an exit handler to ensure we always clean up.
+function cleanup {
+  docker stop $CID
+  docker rm --volumes --force $CID
+}
+trap cleanup EXIT
+
+# Get our host and port ... like 0.0.0.0:32751
+HOST_AND_PORT=`docker port $CID 5432/tcp`
+
+# The postgres user already exists, so we can go ahead and create the database
+until docker exec $CID psql -U postgres -c 'create database gem'
+do
+  echo "waiting for postgres container..."
+  sleep 0.5
+done
+
+# Set up the schema and run tests
+/usr/local/bin/sbt                                        \
```

Comment:
Would it be better to run all sbt tasks in one go to benefit from the JVM being warm?